hrms/payroll/doctype/payroll_entry/payroll_entry.py

```diff
@@ -592,84 +658,105 @@ def get_amount_and_exchange_rate_for_journal_entry(
 		conversion_rate = 1
 		exchange_rate = self.exchange_rate
 		account_currency = frappe.db.get_value("Account", account, "account_currency")
+
 		if account_currency not in currencies:
 			currencies.append(account_currency)
+
 		if account_currency == company_currency:
 			conversion_rate = self.exchange_rate
 			exchange_rate = 1
+
 		amount = flt(amount) * flt(conversion_rate)
 
 		return exchange_rate, amount
 
 	@frappe.whitelist()
-	def make_payment_entry(self):
+	def make_bank_entry(self):
```

Comment:
Renaming a whitelisted method is a breaking change. Since this will be shipped to minor versions too, I don't think we should be doing it here. People might have dependent code on this.